bin/RUV_normalisation_BluePrint.R

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env Rscript
+
+# Author: Suhaib Mohammed
+
+suppressMessages( library( ExpressionAtlas) )
+suppressMessages( library( funr ) )
+
+script_path<-funr::get_script_path()
+
+## loading generic functions 
+source(paste(script_path,"generic_functions.R",sep="/"))
+
+# Get commandline arguments.
+args <- commandArgs( TRUE )
+
+if( length( args ) != 2 ) {
+  stop( "\nUsage:\n\tscript.R <study_list> <output_path>\n\n" )
+}
+
+# Get the transcripts expression filename from the arguments.
+study_list <- args[ 1 ]
```

Comment:
I would prefer that we use a proper argument parsing library here, like optparse